### DISCUSSION TITLE
Windows Container: Path variables inside the container started by `nerdctl run` is not set properly.

### DISCUSSION THREAD

**Discussion by yldoge** [containerd/nerdctl#2285] [Q&A]
Versions:
- nerdctl: 0.21.0
- containerd: 1.6.6
- CNI: 0.3.0

Command:
```
nerdctl run -it <image-id> cmd
```

The PATH inside container: it is a linux path.
![image](https://github.com/containerd/nerdctl/assets/91512011/876de5e8-763f-4430-bf4b-997df0ed6c46)

**Comment by brwilkinson** [comment]
Added issue:
- https://github.com/containerd/nerdctl/issues/2485
